Solution/ContosoProject/Domain/Entities/Comment.cs

```diff
@@ -6,7 +6,52 @@
 
 namespace Domain.Entities
 {
-    public class Comment
+    enum CommentType { Order=1, Customer, Goods}
+
+    public class Comment:BaseEntity
     {
+        private string message;
+        private CommentType type;
+
+        public string Message
+        {
+            get
+            { 
+                return message;
+            }
+            set
+            {
+                if (value.Length != 0)
+                {
+                    message = value;
+                }
+
+                else
+                {
+                    message = "default message";
+                }
+            }
+        }
+
+        public CommentType Type
+        {
+            get
+            {
+                return type;
+            }
+
+            set
```

Comment:
Maybe you need private set or any constraints for this property? It look's like "public CommentType type; ".